Mathlib/Data/Fin/Basic.lean

```diff
@@ -442,6 +442,10 @@ theorem val_add_eq_ite {n : ℕ} (a b : Fin n) :
     Nat.mod_eq_of_lt (show ↑b < n from b.2)]
 --- Porting note: syntactically the same as the above
 
+lemma intCast_val_sub_eq_ite {n : ℕ} (a b : Fin n) :
```

Comment:
```suggestion
lemma intCast_val_sub_eq_sub_add_ite {n : ℕ} (a b : Fin n) :
```
maybe? Otherwise it suggests that the rhs itself is an ite, which is also possible.

bors d+